packages/devtools_app/lib/src/shared/common_widgets.dart

```diff
@@ -642,6 +642,7 @@ class DevToolsTooltip extends StatelessWidget {
     this.padding = const EdgeInsets.all(defaultSpacing),
     this.decoration,
     this.textStyle,
+    this.tooltipTextColor,
```

Comment:
instead of adding an extra parameter for the tooltipTextColor, this can be passed in as part of the text style.

Reply:
unfortunately the text style gets overriden so I would need to refactor things a bit inside the Dev tool tip to make that work.

I originally wanted to change the richtext == null case to copy the style and override, but I wasn't sure if that was too heavy handed

Reply:
it's a bug that we are overriding textStyle when `richMessage == null`. Change that if statement to
`if (style == null && richMessage == null)` and this should fix the problem